tests/scripts/create-dev-cluster.sh

```diff
@@ -123,13 +147,15 @@ show_usage() {
     echo " Usage: $(basename "$0") [-r] [-d /path/to/rook-examples/dir]"
     echo "  -r        Enable rook orchestrator"
     echo "  -m        Enable monitoring"
-    echo "  -d value  Path to Rook examples directory (i.e github.com/rook/rook/deploy/examples)"
+    echo "  -f        enforce creation of the minikube environment "
+    echo "  -d value  Path to Rook examples directory \(i.e github.com/rook/rook/deploy/examples\)"
+    echo "  -y value  Specify minikube driver \(hypervisor\)"
```

Comment:
TBH I don't see the need for this new option unless you really have a use case for it. The current code auto-detect the driver and this script is meant for rook developers (not generic minikube installation). You can find the supported drivers here [minikube](https://rook.io/docs/rook/v1.12/Contributing/development-environment/#minikube)

Reply:
@rkachach  wrote:
> TBH I don't see the need for this new option unless you really have a use case for it. The current code auto-detect the driver and this script is meant for rook developers (not generic minikube installation).

my use case (as a developer is that the auto-detection mechanism does not work for me:  On a mac with Intel processor, I need to use qemu (which works for me, while hyperkit did not work at all when I tried it. So I added the switch to be able to override the auto-detect.

> You can find the supported drivers here [minikube](https://rook.io/docs/rook/v1.12/Contributing/development-environment/#minikube)

as mentioned, hyperkit does not work for me on an intel Mac

Reply:
> @rkachach wrote:
> 
> > TBH I don't see the need for this new option unless you really have a use case for it. The current code auto-detect the driver and this script is meant for rook developers (not generic minikube installation).
> 
> my use case (as a developer is that the auto-detection mechanism does not work for me: On a mac with Intel processor, I need to use qemu (which works for me, while hyperkit did not work at all when I tried it. So I added the switch to be able to override the auto-detect.
> 
> > You can find the supported drivers here [minikube](https://rook.io/docs/rook/v1.12/Contributing/development-environment/#minikube)
> 
> as mentioned, hyperkit does not work for me on an intel Mac

Well, if this is the case then I think we should fix the autodetection and the docs on [minikube](https://rook.io/docs/rook/v1.12/Contributing/development-environment/#minikube) as from these docs a combination of `Mac/Intel` would lead to using `hyperkit`